webapp/packages/core-ui/src/Form/FormState.ts

```diff
@@ -210,32 +156,22 @@ export class FormState<TState> implements IFormState<TState> {
     return this;
   }
 
-  setException(exception: Error | (Error | null)[] | null): this {
-    this.exception = exception;
-    return this;
-  }
-
   setState(state: TState): this {
     this.state = state;
     return this;
   }
 
   async save(): Promise<boolean> {
     try {
-      this.isSaving = true;
       const context = await this.submitTask.execute(this);
 
       if (ExecutorInterrupter.isInterrupted(context)) {
         return false;
       }
 
-      this.exception = null;
       return true;
-    } catch (exception: any) {
-      this.exception = exception;
-    } finally {
-      this.isSaving = false;
-    }
+    } catch (exception: any) {}
```

Comment:
please remove try/catch

Reply:
I don't think so
this is gonna break the logic in around 15 places

Reply:
ah i see, you kept logic as before (not to throw) but now we have possibility of untracked exceptions